content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md

```diff
@@ -0,0 +1,347 @@
+---
+reviewers:
+- sig-cluster-lifecycle
+title: Upgrading kubeadm clusters from v1.13 to v1.14
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+This page explains how to upgrade a Kubernetes cluster created with `kubeadm` from version 1.13.x to version 1.14.x, and from version 1.14.x to 1.14.y, where `y > x`.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+- You need to have a `kubeadm` Kubernetes cluster running version 1.13.0 or later.
+  [Swap must be disabled][swap].
+  The cluster should use a static control plane and etcd pods.
+- Make sure you read the [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md) carefully.
+- Make sure to back up any important components, such as app-level state stored in a database.
+  `kubeadm upgrade` does not touch your workloads, only components internal to Kubernetes, but backups are always a best practice.
+
+
+[swap]: https://serverfault.com/questions/684771/best-way-to-disable-swap-in-linux
+### Additional information
+
+- All containers are restarted after upgrade, because the container spec hash value is changed.
+- You can upgrade only from one minor version to the next minor version.
+  That is, you cannot skip versions when you upgrade.
+  For example, you can upgrade only from 1.10 to 1.11, not from 1.9 to 1.11.
+
+{{< warning >}}
+The command `join --experimental-control-plane` is known to fail on single node clusters created with kubeadm v1.12 and then upgraded to v1.13.x.
+This will be fixed when graduating the `join --control-plane` workflow from alpha to beta.
+A possible workaround is described [here](https://github.com/kubernetes/kubeadm/issues/1269#issuecomment-441116249).
+{{</ warning >}}
```

Comment:
/assign @fabriziopandini 
we technically don't need the above warning anymore for 1.13->1.14?
am i correct?

Reply:
I like the warning for folks who might be concerned.